### PR TITLE
fix(curriculum): test to check the ".canvas" selector

### LINKED ISSUE #45745: Wrong css class is being tested - Beta RWD
## Affected page

https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-the-css-box-model-by-building-a-rothko-painting/step-22

## Your code

```
  padding: 1px;
  overflow: hidden;
```

This code passes without any failed tests.

## Expected behavior

The test should fail because the instructions clearly states "Replace the `padding` property with `overflow`".

However, it does not fail because the test is checking the wrong selector.

> You should remove the padding property from the .one selector.
> 
> ```
> const onePadding = new __helpers.CSSHelp(document).getStyle('.one').getPropertyValue('padding');
> assert(!onePadding);
> ```

[Link to the test](https://github.com/freeCodeCamp/freeCodeCamp/blob/main/curriculum/challenges/english/14-responsive-web-design-22/learn-the-css-box-model-by-building-a-rothko-painting/60a3e3396c7b40068ad6997f.md)

## Proposed changes

The correct selector to check is ".canvas" class. Changing `one` to `canvas` would fix this issue.

## Screenshots

image

## System

- Device: Laptop
- OS: Windows 10
- Browser: firefox
- Version: 99.0.1 (64-bit)

## Additional context

I am working on fixing this.

[Forum post](https://forum.freecodecamp.org/t/not-sure-if-this-is-a-bug-overflow-hidden/505694/6?u=lemon1)

### PR DESCRIPTION
**Pull request by lem0n4id**
Changed the word "one" to "canvas" as the correct selector to check is ".canvas" class.

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #45745

### CODE REVIEW

File: curriculum/challenges/english/14-responsive-web-design-22/learn-the-css-box-model-by-building-a-rothko-painting/60a3e3396c7b40068ad6997f.md [hunk 1]
```diff
@@ -13,10 +13,10 @@ Replace the `padding` property with `overflow` set to `hidden` - changing the ca
 
 # --hints--
 
-You should remove the `padding` property from the `.one` selector.
+You should remove the `padding` property from the `.canvas` selector.
 
 ```js
-const onePadding = new __helpers.CSSHelp(document).getStyle('.one').getPropertyValue('padding');
+const onePadding = new __helpers.CSSHelp(document).getStyle('.canvas').getPropertyValue('padding');
 assert(!onePadding);
```
Comment (gikf) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-the-css-box-model-by-building-a-rothko-painting/60a3e3396c7b40068ad6997f.md:20]: Small nitpicking, it'd be nice to have `onePadding` name changed as well for consistency.

Comment (lem0n4id) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-the-css-box-model-by-building-a-rothko-painting/60a3e3396c7b40068ad6997f.md]: Done!

**Review by gikf — APPROVED** [review]
Looks good! :+1:

**Review by lasjorg — APPROVED** [review]
LGTM

**Pull request merged**